### PR TITLE
Representative List UI

### PR DESCRIPTION
**Pull request by jindaln**
Added styling for the Rep List

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -430,36 +430,83 @@ function goBack() {
...
+}
+
+//Return questionnaire inner div
+function questionnaireInnerDiv(){
+    var innerDiv = document.createElement("div");
+    innerDiv.className = "col-sm-10";
+    return innerDiv;
+}
+
+
```
Comment (jynCoder) [inline src/main/webapp/script.js]: nit: unnecessary blank line

File: src/main/webapp/repList.html [hunk 2]
```diff
@@ -1,12 +1,16 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
+<head>
     <title>Representative List</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <link rel="stylesheet" href="style.css">
```
Comment (jynCoder) [inline src/main/webapp/repList.html:7]: maybe unnecessary stylesheet doesnt seemed to be used

Comment (swetha-gangu) [reply src/main/webapp/repList.html:7]: ^

File: src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java [hunk 3]
```diff
@@ -55,7 +56,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             throw new ServletException("Error: " + e.getMessage(), e);
         }
         if (rep == null) {
-            DatastoreManager.insertRepresentativeInDatastore(repName, title, username, password);   
+            List<Long> tabIds = DatastoreManager.insertTabsInDatastore(
+                new ArrayList<String> (Arrays.asList(repName.replaceAll("\\s+","") + "Other")), 
+                new ArrayList<String> (Arrays.asList("")));
```
Comment (swetha-gangu) [inline src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java:61]: Could you explain how this function works real quick?

**Comment by jindaln** [comment]
Since I merged PR's out of order, the bulk of repListUI is already in master hence the few code changes

File: src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java [hunk 4]
```diff
@@ -56,11 +56,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             throw new ServletException("Error: " + e.getMessage(), e);
         }
         if (rep == null) {
+            //Initializes rep's tab list with an Other tab prepended by repName for uniqueness
             List<Long> tabIds = DatastoreManager.insertTabsInDatastore(
                 //Prepend the Other tag with repName for uniqueness and 
                 // remove spaces in repName with replaceAll
                 Arrays.asList(repName.replaceAll("\\s+","") + "Other"), 
-                Arrays.asList(""));
+                Arrays.asList("This is my Other tab for all the
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java:64]: make this a constant

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -394,13 +394,10 @@ function displayRepListUser(title, name, inDatastore, image) {
     var imageElement = document.createElement("img");
     imageElement.className = "w3-bar-item w3-circle w3-hide-small";
     imageElement.style = "width:85px";
+    imageElement.src = (inDatastore) ? image : "/images/defaultProfilePicture.png";
```
Comment (v1shanker) [inline src/main/webapp/script.js:397]: nit: don't need parens around inDataastore

**Pull request merged**